src/main/webapp/js/query.js

```diff
@@ -0,0 +1,220 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+let responseJson;
+
+function query() {
+    const errorEl = document.getElementById("error");
+    let lat = localStorage.getItem("lat");
+    let lon = localStorage.getItem("lng");
+    const radius = $('#radius').val();
+    const searchTerms = document.getElementById('searchTerms').value;
+    saveSearch(lat, lon, radius, searchTerms);
+    fetch(`/query`, { method: 'GET' })
+        .then(response => response.json())
+        .then((response) => {
+            responseJson = response; // Debug
```

Comment:
what is the //Debug comment referring to?

Reply:
removed